doc/org.eclipse.cdt.doc.user/src/asciidoc/example.adoc

```diff
@@ -0,0 +1,41 @@
+////
+Copyright (c) 2024 John Dallaway and others
+This program and the accompanying materials
+are made available under the terms of the Eclipse Public License 2.0
+which accompanies this distribution, and is available at
+https://www.eclipse.org/legal/epl-2.0/
+
+SPDX-License-Identifier: EPL-2.0
+
+Contributors:
+    John Dallaway - initial content
+////
+
+// support image rendering and table of contents within GitHub
+ifdef::env-github[]
+:imagesdir: https://raw.githubusercontent.com/eclipse-cdt/cdt/main/doc/org.eclipse.cdt.doc.user/images
+:toc:
+:toc-placement!:
```

Comment:
We need these github-specific directives to support presentation of a table of contents in github.